src/charts/line/line.js

```diff
@@ -1107,6 +1109,17 @@ export default function module() {
             : d1;
     }
 
+    /**
```

Comment:
Shouldn't this be above findOutNearestDate?

Reply:
Oops, you are right! I will change it!

Thanks!